launcher/modManager/cdownloadmanager_moc.cpp

```diff
@@ -10,7 +10,7 @@
 #include "StdInc.h"
 #include "cdownloadmanager_moc.h"
 
-#include "../launcherdirs.h"
+#include "vcmiqt/launcherdirs.h"
```

Comment:
external includes should use angle brackets
```suggestion
#include <vcmiqt/launcherdirs.h>
```

Reply:
For vcmi library we use form like `"../lib/header.h"` - relative path & double-quotes